src/containerapp/azext_containerapp/_params.py

```diff
@@ -66,6 +66,10 @@ def load_arguments(self, _):
         c.argument('managed_certificates_only', options_list=['--managed-certificates-only', '-m'], help='List managed certificates only.')
         c.argument('private_key_certificates_only', options_list=['--private-key-certificates-only', '-p'], help='List private-key certificates only.')
 
+    with self.argument_context('containerapp env dapr-component init') as c:
+        c.argument('statestore', help="The state store component and dev service to create.")
```

Comment:
The argument is `statestore`?  Recommend to us: `state-store`.

Reply:
Actually, this is consistent with Dapr conventions. For example, see component name in https://docs.dapr.io/operations/components/setup-state-store/